Add test data to Docker development environment

redshiftzero opened the pull request:
## Status

Ready for review

## Description of Changes

Closes #1662, adds some test sources, submissions and a non-admin user to the docker dev env

## Testing

```
make dev
```

Login as both `journalist` and `admin` and check that a couple sources are loaded:

```
Test user successfully added: username=journalist, password=WEjwn8ZyczDhQSK24YKM8C9a, otp_secret=JHCOGO7VCER3EJ4L, is_admin=False
Test user successfully added: username=admin, password=WEjwn8ZyczDhQSK24YKM8C9a, otp_secret=JHCOGO7VCER3EJ4L, is_admin=True
Test source 'uninvolved payoff' added with 2 submissions
Test source 'valuable workbook' added with 2 submissions
```

## Deployment

None, dev env only

## Checklist

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### If you made changes to documentation:

- [x] Doc linting passed locally

dachary commented:
<pre>
Test source 'hoary saxifrage' added with 2 submissions
Test source 'verdant quietist' added with 2 submissions
</pre>
Is ok but when I login locahost:8081 as a journalist, I don't see them.

Would it be possible to keep the journalist as admin so the demo page does not have to be changed? The and maybe call the journalist with no admin privileges "journalist-not-admin" or something?

dachary commented:
Great idea by the way :-) I wanted that for the longest time and you did it! :+1:

redshiftzero commented:
Ah sorry, forgot to add `Source.pending`, fixed. And re the username, sure why not - I've updated the second username to be the same as in our [test data](https://github.com/freedomofpress/securedrop/blob/7a4af2ef04a096ef5b337e267cab621e802ecb39/securedrop/tests/test_journalist.py#L448) 🙃
